github/lib/fetch_repos.rb

```diff
@@ -0,0 +1,20 @@
+require "octokit"
```

Comment:
We don't use Octokit in this class (other than what we pass in), but we do call `YAML.load_file`:

```suggestion
require "yaml"
```